src/components/Footer.tsx

```diff
@@ -66,14 +82,17 @@ const Footer: React.SFC<FooterProps> = () => {
                 <Link to="/pricing#enterprise">Enterprise</Link>
                 <a href="https://status.gitpod.io" target="_blank">Status</a>
             </FooterMenu>
-            <div style={{ height: 40, display: 'flex', fontSize: 10, color: colors.fontColor1 }}>
+            <SocialLinks>
                 <p style={{padding: 10}}>Stay connected</p>
                 <a href="https://github.com/gitpod-io/gitpod"><SocialImg alt="GitHub Icon" src={github}/></a>
                 <a href="https://spectrum.chat/gitpod"><SocialImg alt="Spectrum Icon" src={spectrum}/></a>
                 <a href="https://twitter.com/gitpod"><SocialImg alt="Twitter Icon" src={twitter}/></a>
-            </div>
-            <div style={{ height: 40, marginTop: 15, display: 'flex', fontSize: 10, color: colors.fontColor1 }}>
-                Copyright © 2019&nbsp;<a href="https://typefox.io" target="_blank">TypeFox</a>&nbsp;| All Rights Reserved |&nbsp;<Link to="/imprint">Imprint</Link>&nbsp;|&nbsp;<Link to="/privacy">Privacy Policy</Link>&nbsp;|&nbsp;<Link to="/terms">Terms of Service</Link>
+            </SocialLinks>
+            <div style={{ height: 40, fontSize: 13, color: colors.fontColor1, textAlign: 'center' }}>
```

Comment:
I think that just a little bit more space at the bottom would be nice.

```suggestion
            <div style={{ height: 50, fontSize: 13, color: colors.fontColor1, textAlign: 'center' }}>
```